qubes/mgmt.py

```diff
@@ -112,6 +112,12 @@ def __init__(self, app, src, method, dest, arg):
         #: name of the method
         self.method = method.decode('ascii')
 
+        #: callback for sending events if applicable
+        self.send_event = send_event
+
+        #: is this operation cancellable?
+        self.cancellable = False
+
```

Comment:
We may want to write that down somewhere. This is probably wise, so long operation don't get interrupted and result for example in some inconsistent filesystem state.